methods/array/filter.js

```diff
@@ -0,0 +1,19 @@
+function filter(array, Function) {
```

Comment:
wow, this a keyword. Please, don't use keywords as a name of variable. You can name it `callback`